pkg/console/subresource/configmap/managed_cluster_oauth_server_cert.go

```diff
@@ -0,0 +1,41 @@
+package configmap
+
+import (
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+
+	"github.com/openshift/console-operator/pkg/api"
+	"github.com/openshift/console-operator/pkg/console/subresource/util"
+)
+
+func DefaultManagedClusterOAuthServerCertConfigMap(clusterName string, caBundle string, cr *operatorv1.Console) *corev1.ConfigMap {
+	configMap := ManagedClusterOAuthServerCertConfigMapStub(clusterName)
+
+	if caBundle != "" {
```

Comment:
does it make sense to call `DefaultManagedClusterOAuthServerCertConfigMap` function if the `caBundle` is empty?

Reply:
I'll remove this empty string check. We check for empty string before calling this, so it should never be empty.